resources/views/partials/analytics.blade.php

```diff
@@ -18,3 +18,19 @@
   _gs('{{ $gosquaredTracking }}');
 </script>
 @endif
+@if($piwikTracking = Setting::get('app_analytics_pi_url'))
```

Comment:
Since there is a default value (1) in the form (see <https://github.com/ldidry/Cachet/commit/78639072c6e921c78ea166464cedcb8cc321bbc1#diff-a45ca592db654261f0e7da8c28be27d2R74>), when you save your setup, you set `app_analytics_pi_siteid`, even if you don't set `app_analytics_pi_url`.

So I think we don't need to check `app_analytics_pi_siteid`, but I can add a check if you think it's relevant.